src/tools/rbd/action/ImageMeta.cc

```diff
@@ -98,6 +98,8 @@ static int do_metadata_remove(librbd::Image& image, const char *key)
 {
   int r = image.metadata_remove(key);
   if (r < 0) {
+    if( r== -ENOENT)
```

Comment:
Nit: move this outside the nested if so that it doesn't print two error messages